TSOMessageHub/Jobs/SendXML.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Quartz;
+using System.Xml.Serialization;
+using MassTransit;
+using TSOMessageHub.XML;
+
+namespace TSOMessageHub;
+
+public class SendXML : IJob
+{
+    private readonly IPublishEndpoint _publishEndpoint;
+    private readonly char _separator = Path.DirectorySeparatorChar;
+    private readonly string _projectPath = new Uri(Directory.GetCurrentDirectory()).LocalPath;
+    private readonly static Random _random = new Random();
+    private static decimal _quantityMw = 0;
+    private static int _counter = 0;
+    private readonly TSOSignal _signal;
+
+    public SendXML(ILogger<SendXML> logger, IPublishEndpoint publishEndpoint)
```

Comment:
Remove logger from parameters since _logger field was already removed
```suggestion
    public SendXML(IPublishEndpoint publishEndpoint)
```

Reply:
how did I miss it...